README.md

```diff
@@ -1,58 +1,31 @@
 [![Chat](https://img.shields.io/discord/457912077277855764?label=chat&logo=discord)](https://svelte.dev/chat)
 
-# READ THIS FIRST!
+# SvelteKit
 
-SvelteKit is in its release candidate phase:
-- :exclamation: expect bugs! 
-- :heavy_check_mark: no more planned breaking changes
-- :heavy_check_mark: countdown to a stable release
-- :x: feature-complete
-- :x: tutorials
-
-Watch [the announcement](https://www.youtube.com/watch?v=A8jkJTWacow&t=29628s) and track [progress towards 1.0](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0) for more details.
-
-## Overview
-
-The Fastest Way to Build Svelte Apps
-
-- 💨 Blazing-Fast Production Sites
-- 🛠️ SSR, SPA, SSG, and In-Between
-- ⚡️ Instantly Visible Code Changes
-- 🔩 Existing Universe of Plugins
-- 🔑 Fully Typed APIs
-
-## Documentation
-
-Please see [the documentation](https://kit.svelte.dev/docs) for information about getting started and developing with SvelteKit.
+Web development, simplified. Read the [documentation](https://kit.svelte.dev/docs) to get started.
```

Comment:
we have "streamlined"  as the tagline on the home page - should we align this?

Reply:
dammit that's what i meant to write. sleep deprivation

```suggestion
Web development, streamlined. Read the [documentation](https://kit.svelte.dev/docs) to get started.
```